src/rules/color-hex-length/index.js

```diff
@@ -0,0 +1,48 @@
+import {
+  report,
+  ruleMessages,
+  styleSearch
+} from "../../utils"
+
+export const ruleName = "color-hex-length"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (l, h) => `Expected ${l} hex color ${h}`,
```

Comment:
If I were developing this rule would be frustrating to me if it didn't inform me of the *expected* notation. Something like `Expected short hex color notation ${short}`. Does that make sense? Then I would immediately know what to change it to.